content/docs/clients/python.mdx

```diff
@@ -3,143 +3,62 @@ title: Python
 description: How to install and use Ada in a Python project
 ---
 
-## Requirements
+## Installing pre-built binary wheels
 
-The project is self-contained and has no dependency.
-A recent C++ compiler supporting C++17. We test GCC 9 or better, LLVM 10 or better and Microsoft Visual Studio 2022.
+The easiest way to get started is to install `ada-url` from [PyPI](https://pypi.org/project/ada-url/).
 
-## Building
+From within your virtual environment, run:
 
-Ada uses **cmake** as a build system. It's recommended to have cmake available in your system.
-Run the following commands to compile and build Ada locally.
+```bash
+python3 -m pip install ada-url 
+```
 
-<Steps>
+## Installing from source
 
-  ### Prepare
+The `Makefile` in the [project root](https://github.com/ada-url/ada-python) will build a package for installation on your own machine.
 
-  ```bash
-  cmake -B build
-  ```
+From within your virutal environment, run:
 
-  ### Build
-
-  ```bash
-  cmake --build build
-  ```
-
-</Steps>
+```bash
```

Comment:
Can you convert this into Steps like in: https://github.com/ada-url/website/blob/main/content/docs/installation.mdx?plain=1#L41